src/main/java/org/gwtbootstrap3/extras/slider/client/ui/base/SliderBase.java

```diff
@@ -346,6 +346,20 @@ public void setTicks(final List<Double> ticks) {
         updateSliderForNumberArray(SliderOption.TICKS, ticks);
     }
 
+    public List<String> getTicksPositions() {
+        return getStringArrayAttribute(SliderOption.TICKS_POSITIONS, Collections.<String>emptyList());
+    }
+
+    /**
+     * Defines the positions of the tick values in percentages.<br>
+     * The first value should always be 0, the last value should always be 100 percent.
+     *
+     * @param ticksPositions
+     */
+    public void setTicksPositions(final List<String> ticksPositions) {
+        updateSliderForStringArray(SliderOption.TICKS_POSITIONS, ticksPositions);
+    }
+
```

Comment:
According to the API doc, the new ```tick_position``` option should be numeric value. So please change the argument type to ```List<Double>``` just like the ```ticks```options.